website/docs/user_guide/projects.md

```diff
@@ -3,13 +3,17 @@ id: projects
 title: Projects
 ---
 
-<div class="alert alert--info" role="alert">
-  Projects are part of Unleash Pro and Unleash Enterprise.
-</div>
+:::info feature availability
 
-## Overview {#overview}
+All users get access to Unleash's projects, but only pro and enterprise users can create, update, or delete them.
```

Comment:
I actually have no idea who the projects belong to here from either a linguistic or software perspective. Since we're already in a section called projects in the Unleash docs, we can avoid this entire problem but just trimming the word out. Entirely optional!

```suggestion
All users get access to projects, but only pro and enterprise users can create, update, or delete them.
```